meta-iotqa/lib/oeqa/runtime/peripherals/mraa/mraa_i2c.py

```diff
@@ -0,0 +1,20 @@
+from oeqa.oetest import oeRuntimeTest
+
+
+class MraaI2cTest(oeRuntimeTest):
+    def test_mraa_i2c_version(self):
+        (status, output) = self.target.run("mraa-i2c version")
+        self.assertEqual(status, 0, msg="Error messages: %s" % output)
+
+    def test_mraa_i2c_list(self):
+        (status, output) = self.target.run("mraa-i2c list")
+        self.assertEqual(status, 0, msg="Error messages: %s" % output)
+
+    def test_mraa_i2c_detectbus(self):
+        (status, output) = self.target.run("mraa-i2c detect bus")
+        self.assertEqual(status, 0, msg="Error messages: %s" % output)
+
+    def test_i2c_value(self):
+        (status, output) = self.target.run("mraa-i2c set 1 0x02 0x03 0x04 > /tmp/log.txt")
```

Comment:
please add a comment to explain the purpose of this test and how the bus address, register, and value got chosen